chemprop/train/molecule_fingerprint.py

```diff
@@ -13,13 +13,16 @@
 from chemprop.models import MoleculeModel
 
 @timeit()
-def molecule_fingerprint(args: FingerprintArgs, smiles: List[List[str]] = None) -> List[List[Optional[float]]]:
+def molecule_fingerprint(args: FingerprintArgs,
+                         smiles: List[List[str]] = None,
+                         return_invalid_smiles: bool = True) -> List[List[Optional[float]]]:
```

Comment:
I don't know why the default of `return_invalid_smiles` in [predict_and_save](https://github.com/chemprop/chemprop/blob/98fc25eec8c57171c56d6ffad03ae1336c016e3b/chemprop/train/make_predictions.py#L135) is `False`. In my opinion, those two values should be the same due to consistency. The reason I believe that the default of `True` would make sense in [molecule_fingerprint](https://github.com/chemprop/chemprop/blob/master/chemprop/train/molecule_fingerprint.py#L18) is that it would only affect the results returned by `molecule_fingerprint()`. This returned value is not used anywhere when using CML running Chemprop, so the default only affects those who use the API directly. Without setting the default to `True`, the results cannot be mapped back to the original SMILES list, so I think using the default of `True` would make more sense in this case.